### DISCUSSION TITLE
installed but not open raspberry os

### DISCUSSION THREAD

**Discussion by osgoodtr** [mcguirepr89/BirdNET-Pi#1071] [Q&A]
hi

i have P4 4GB. i use Raspberry Pi OS (Legacy, 64-bit) bulleye. i follow the Installation Guide. step 20 after system automatically reboot but not open any program. its wait like this(add photo)
![Ekran Resmi 2023-11-10 20 59 22](https://github.com/mcguirepr89/BirdNET-Pi/assets/51755503/e89474c2-c3b5-4872-9734-48b8510af6e0)

**Comment by Svardsten53** [comment]
You have to open the program in a web browser: [http://birdnetpi.local](url)

**Reply by osgoodtr** [reply]
ım sorry. how i open web browser ? what i white ?
